planning/behavior_path_planner/src/utils/goal_planner/freespace_pull_over.cpp

```diff
@@ -77,6 +76,7 @@ boost::optional<PullOverPath> FreespacePullOver::plan(const Pose & goal_pose)
 
   // remove points which are near the goal
   PathWithLaneId & last_path = partial_paths.back();
+  // todo: define as parameter
```

Comment:
either is fine.

just hard code like
https://github.com/autowarefoundation/autoware.universe/blob/b16ab344870b9d6fd6cbb471f67b4acaa958e071/planning/behavior_path_planner/src/utils/path_utils.cpp#L71-L73

Reply:
I removed this comment in https://github.com/autowarefoundation/autoware.universe/pull/4804/commits/5a8ea3575527144c54b51989cb5f372b4dd7c34e